pages/common/yesod.md

```diff
@@ -0,0 +1,19 @@
+# yesod
+
+> Yesod web framework CLI helper.
+
+- Install Yesod CLI tool in a Yesod scaffolding:
+
+`stack build yesod-bin cabal-install --install-ghc`
+
+- Start development server:
+
+`stack -- exec yesod devel`
+
+- Touch files with altered Template Haskell dependencies:
+
+`stack -- exec yesod devel`
+
+- Deploy application using Keter:
```

Comment:
Think of this description from a newcomer's perspective(like me). I have no idea what Keter is. Please describe that within parentheses.